assessment.md

```diff
@@ -56,43 +56,79 @@ Replace TODOs with abbreviations, or emoji, or ...
 
 * Calculating code coverage of a `pytest` test suite
 
-  TODO: add description for each assessment level for this skill.
+  * :flushed: = Does not attempt to implement any test features with pytest.
+
+  * :worried: = Low code coverage due to inefficient coding.
+
+  * :neutral_face: = Having average code coverage and making attempt to improve code coverage by efficient coding.
+
+  * :blush: = Having high code coverage on a fairly regular basis and knowing how to further improve code coverage.
+
+  * :laughing: = Always having code coverage, ensuring maximum software efficiency.
 
 * Reporting code coverage through an online provider
 
-  TODO: add description for each assessment level for this skill.
+  * :flushed: = Not knowing what code coverage is or how to use online provider to report code coverage.
+
+  * :worried: = Knowing how to report code coverage but not doing it very often
+
+  * :neutral_face: = Knowing how to report code coverage and report at a fairly regular basis.
+
+  * :blush: =  Reporting code coverage at a regular basis and making attempts to improve code coverage through efficient coding.
+
```

Comment:
Replacement suggestion, instead of `efficient coding` a better description would be `using thorough test suite`